agent-zero/umbrel-app.yml

```
@@ -0,0 +1,36 @@
+manifestVersion: 1
+id: agent-zero
+name: Agent Zero
+tagline: A personal, organic agentic framework that grows and learns with you
+category: ai
+version: "0.9.7"
+port: 3005
+description: >-
+  Agent Zero is a personal, organic agentic framework that grows and learns with you.
+  It is fully Dockerized and includes features like Speech-to-Text and TTS.
+
+  **Key Features:**
+  - **General Purpose Assistant:** Can handle various tasks.
+  - **Computer Use:** Can use the computer (in the container).
+  - **Coding:** Can write and execute code.
+  - **Memory:** Learns from interactions.
+
+  **Getting Started:**
+  1. Install Ollama from the Umbrel App Store.
+  2. Launch Agent Zero.
+  3. Configure your models in the settings.
+
+  ⚠️ **Warning:** Agent Zero can execute code. Run with caution.
+developer: Agent Zero AI
+website: https://github.com/agent0ai/agent-zero
+submitter: Agent Zero Community
```

Add yourself as submitter :-) 
```suggestion
submitter: slimatic
```